src/RcppExports.cpp

```diff
@@ -2,6 +2,7 @@
 // Generator token: 10BE3573-1514-4C36-9D1C-5A225CD40393
 
 #include <Rcpp.h>
+#include <sys/stat.h>
```

Comment:
This file is automatically generated by Rcpp, this include will be overwritten next time `Rcpp::compileAttributes()` is run, it cannot go here.